documentation/getting-started/README.md

```diff
@@ -56,51 +63,49 @@ TestCafe Studio displays the recorded test in the [Test Editor](../user-interfac
 
 ![The recorded test](../../images/getting-started/recorded-test.png)
 
-Click an action in the editor to view or change its parameters.
-For example, you can change the selector that identifies the action's element by selecting an alternative selector from the list of generated selectors or using your own.
+The editor allows you to modify the test: reorder and delete actions, add new actions, modify actions' parameters. For example, you can change the [Element Selector](../guides/record-tests/test-actions/on-page-actions/action-parameters.md#element-selector) parameter that specifies the action's target element: select an alternative selector from auto-generated selectors or enter your own.
```

Comment:
The editor allows you to modify the test: reorder and delete actions, add new actions, modify actions' parameters. -> The editor allows you to reorder and delete actions, add new actions, and modify actions' parameters.